src/index.html

```diff
@@ -5,8 +5,271 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header header__wrapper">
+      <a href="#">
+        <img
+          src="./images/logo.png"
+          alt="logo that consists of 2 blue and 2 black squares with white letters in them"
+          class="logo header__logo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__item">
+            <a href="#" class="nav__link is-active">Apple</a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Samsung </a>
+          </li>
+          <li class="nav__item">
+            <a href="#" class="nav__link">Смартфоны и телефоны</a>
+          </li>
+          <li class="nav__item">
+            <a
+              href="#"
+              class="nav__link"
+              data-qa="nav-hover">Ноутбуки и компьютеры
+            </a>
```

Comment:
текст також потрібно переносити 
```suggestion
            <a
              href="#"
              class="nav__link"
              data-qa="nav-hover"
            >
              Ноутбуки и компьютеры
            </a>
```